### PR TITLE
Added the bootstrap documentation

### PR DESCRIPTION
**Pull request by ravitb**

### CODE REVIEW

**Review by chaseeb — COMMENTED** [review]
Do you think it would be beneficial for these docs to tell the options for:

- Horizon_Endpoint
- Network_Passphrase
- Network_Name

Also I do not think it's completely clear to someone how to actually run this server on somewhere like heroku or AWS.  Do we think it might be helpful?  Someone that wants to use the bootstrap is mostly likely doing so because they are unfamiliar with building a node app.  Which also means they are probably unfamiliar with deploying a docker on a server.  It seems like we could provide additional information on the expectations.

File: website/sidebars.json [hunk 1]
```diff
@@ -46,7 +46,8 @@
     ],
     "Node.js": [
       "nodejs/hi-kin",
-      "nodejs/sdk"
+      "nodejs/sdk",
+      "python/bootstrap"
```
Comment (yosriz) [inline website/sidebars.json]: wrong path

File: docs/nodejs/bootstrap.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+| CHANNEL_STARTING_BALANCE                | Initial Kin balance of each channel                                                                                                                        |
+| CHANNEL_SALT             | A string to be used when creating the channels                                                                                                                                                      |
+| PORT                | Port to serve the app on                                                                                                                                                                                                             |
+| LOG_LEVEL             | Log level, either "INFO" or "ERROR"                                                                                                                                                      |
+
+
+## Running the Server
+```bash
```
Comment (yosriz) [inline docs/nodejs/bootstrap.md:33]: must we clone the repo, right? if so, add this step.
In general, I think we need here 0 to 1 steps including how to generate a seed, and default values inside the docker-compose for reference. the idea is to have dead-simple instruction in one place.
I understand that it's borrowed from python bootstrap, so adding also @Ronserruya to get his input.

File: docs/nodejs/bootstrap.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+A ready-to-use server using the [Kin Node.js SDK](https://github.com/kinecosystem/kin-sdk-node/tree/master)
+
+## Prerequisites
+Docker software installed:
+* [Docker](https://github.com/docker/docker-install)
+* [docker-compose](https://docs.docker.com/compose/install)
+
+## Configuration
+The server can be configured from the docker-compose.yml file.
```
Comment (yosriz) [inline docs/nodejs/bootstrap.md:14]: fix to .env file, include it in the steps instructions. (see next comment)

**Review by yosriz — CHANGES_REQUESTED** [review]
see comments

**Review by mrsegev — APPROVED** [review]

**Pull request closed without merging**